docs/deploying.md

```diff
@@ -26,7 +26,7 @@ integration with other Jupyter tools.
 To get the [Python CLI](./cli.ipynb) and [API](./api/index.md) from [PyPI]:
 
 ```bash
-pip install jupyterlite
+python3 -m pip install --pre jupyterlite
```

Comment:
Probably some distributions are still on Python 2 by default and require the explicit `python3`?

Otherwise I have a minor preference for just `python` since this is user-facing:

```suggestion
python -m pip install --pre jupyterlite
```